src/components/VContentSwitcher/VSearchTypeButton.vue

```diff
@@ -13,8 +13,8 @@
   >
     <VIcon :icon-path="icon" />
     <span
-      class="md:block md:truncate md:text-left md:ms-2"
-      :class="isHeaderScrolled ? 'hidden' : 'block truncate text-left ms-2'"
+      class="md:block md:truncate md:ms-2 md:text-start"
+      :class="isHeaderScrolled ? 'hidden' : 'block truncate ms-2 text-start'"
```

Comment:
Good catch, surprised the change didn't require updating snapshot tests though :thinking:

Reply:
I caught myself using `left` and `right` classes a couple of days ago 🙈